compliance_checker/cfutil.py

```diff
@@ -307,10 +307,10 @@ def get_forecast_metadata_variables(ds):
     :param netCDF4.Dataset ds: An open netCDF4 Dataset.
     :rtype: list
     '''
-    forecast_metadata_standard_names = [
+    forecast_metadata_standard_names = {
         'forecast_period',
         'forecast_reference_time',
-    ]
+    }
     forecast_metadata_variables = []
```

Comment:
I actually meant `forecast_metadata_variables` for the list -> set conversion, apologies if this wasn't explicit.   This likely to be fairly minor, so I'll just keep as is and merge, possibly revisiting this for #727 and other similar places in code which use lists.